### PR TITLE
Slightly rephrase content of doxygen main page

### PR DESCRIPTION
**Pull request by mayrmt**
## Description and Context

I merged #108 prematurely, so this PR delivers changes that came up during review of #108.

## Related Issues and Merge Requests

Follows #108.

### CODE REVIEW

File: doc/doxygen/index.doc [hunk 1]
```diff
@@ -2,19 +2,21 @@
...
 \endhtmlonly
 
-\section Overview
+\section mainpageoverview Overview
 
 4C (Comprehensive Computational Community Code) is a powerful research code for multi-physics computer simulations.
 Building on more than 20 years of research and development, 4C can model a plethora of physical systems, including solid mechanics, fluid dynamics, scalar transport, and chemical reaction among others.
 
-Large parts of 4C are based on finite element methods (FEM), but alternative discretization methods such as discontinuous Galerkin methods (DG), particle methods and mesh-free methods have also been successfully integrated. The research software is implemented throughout in object-oriented programming (C++) using modern software design and is parallelized with MPI for distributed memory hardware architectures.
+Large parts of 4C are based on finite element methods (FEM), but alternative discretization methods such as discontinuous Galerkin methods (DG), particle methods and mesh-free methods have also been successfully integrated. The research software is implemented throughout in C++ using modern software design and is parallelized with MPI for distributed memory hardware architectures.
```
Comment (sebproell) [inline doc/doxygen/index.doc:10]: These sentences always sounded rather German to me. How about this:

```suggestion
4C mainly implements finite element methods (FEM) but also offers alternative discretization methods such as discontinuous Galerkin methods (DG), particle methods, and mesh-free methods. 4C is written in C++ using modern software design and uses MPI for distributed memory hardware architectures.
```

**Review by sebproell — APPROVED** [review]

**Review by amgebauer — APPROVED** [review]

**Comment by mayrmt** [comment]
When looking at the artifacts, the main page does not show any content, though I don't understand why.

File: doc/doxygen/index.doc [hunk 2]
```diff
@@ -2,19 +2,21 @@
...
 This document / web page provides source code documentation via Doxygen. It documents code constructs such as classes, functions, or class members.
 
 A high-level documentation of 4C (including a user guide and a developer guide) can be found in our user manual: https://4c-multiphysics.github.io/4C/readthedocs/
 
-\section Further resources
+\section mainpagepurposefurtherresources Further resources
 
+- Website: https://www.4c-multiphysics.org
 - Source Code Repository on GitHub: https://github.com/4C-multiphysics/4C
+- Readthedocs: https://4c-multiphysics.github.io/4C/readthedocs/
```
Comment (amgebauer) [inline doc/doxygen/index.doc:22]:
```suggestion
- Readthedocs: https://4c-multiphysics.github.io/4C/readthedocs/
*/
```

I think this has been accidentally removed which is necessary that doxygen interprets the content

**Review by amgebauer — CHANGES_REQUESTED** [review]

**Review by amgebauer — APPROVED** [review]

**Comment by mayrmt** [comment]
@amgebauer, thanks for the hint! Main page looks good now.

# Summary

This PR updates some text on the main page and adds some more links to further resources.

**Review by sebproell — APPROVED** [review]
I would squash the commits. Unfortunately, we cannot enable the automatic option on Github, since it would perform a fast-forward merge if possible (w/o a merge commit).

**Comment by mayrmt** [comment]
@sebproell @amgebauer I have squared manually. Please reapprove.

**Pull request merged**